### PR TITLE
feat: generate cjs and esm atom exports

### PR DESCRIPTION
**Pull request by axyz**
@sylvesteraswin 

- generate `index.js` (cjs) and `index.esm.js` (es modules) in each atom
- add snapshots
- update templates
- remove dash classes

### CODE REVIEW

File: integration-test/packages/atoms/for/index.css [hunk 1]
```diff
@@ -28,11 +28,11 @@
         border: $(j)px red solid;
       }
 
-      .another-child_$(i)_$(j) {
+      .anotherChild_$(i)_$(j) {
         width: $(j)em;
       }
 
-      &.another-modifier_$(i)_$(j) {
+      &.anotherModifier_$(i)_$(j) {
```
Comment (sylvesteraswin) [inline integration-test/packages/atoms/for/index.css:35]: How will this effect us?

**Comment by axyz** [comment]
:+1:

File: integration-test/packages/atoms/for/index.css (hunk 1 above)
Comment (axyz) [reply integration-test/packages/atoms/for/index.css:35]: it won't we don't use dashed classes. Ideally we should forbid them breaking the build with an helpful error message

**Comment by sylvesteraswin** [comment]
👍

**Pull request merged**